searx/templates/simple/base.html

```diff
@@ -19,14 +19,7 @@
   {% endif %}
   {% block styles %}{% endblock %}
   <!--[if gte IE 9]>-->
```

Comment:
I assume we can drop the IE9 quirks `<!--[if gte IE 9]>-->`  .. / Microsoft has discouraged the use of Internet Explorer as a default browser.